components/serializer.rst

```diff
@@ -885,6 +885,18 @@ The array keys beginning with ``@`` are considered XML attributes::
     //     <foo bar="value" />
     // </response>
 
+If you want create a node with value and attribute use this::
+
+    array('foo' => array('@bar' => 'value', # => 'baz'));
```

Comment:
minor syntax issue: `# => 'baz'` should be `'#' => 'baz'`

Reply:
agree with you, thanks